src/main/java/blusunrize/immersiveengineering/common/blocks/IETileProviderBlock.java

```diff
@@ -468,6 +469,8 @@ public int getRenderColour(BlockState state, @Nullable IBlockReader worldIn, @Nu
 		return 0xffffff;
 	}
 
+	//TODO BlockFaceShape Gone, commented out for main author deletion.
```

Comment:
`IFaceShape` seems to be the only "interesting" case, and I'd guess that MC can figure that out by itself as well now.

Reply:
Ok, I left the commented method in the code, with the removal marker for the porting-cleanup.